calico_containers/tests/unit/datastore_test.py

```diff
@@ -52,8 +50,17 @@
                                           "1234/"
 TEST_CONT_PATH = CALICO_V_PATH + "/host/TEST_HOST/workload/docker/1234/"
 CONFIG_PATH = CALICO_V_PATH + "/config/"
-BGP_NODE_DEF_AS_PATH = CALICO_V_PATH + "/config/bgp_as"
-BGP_NODE_MESH_PATH = CALICO_V_PATH + "/config/bgp_node_mesh"
+
+BGP_V_PATH = "/calico/bgp/v1"
```

Comment:
I'm sure there's a good reason for it, but why do we have to duplicate all these constants here?